src/main/resources/static/Inscription_mentor.html

```diff
@@ -75,11 +75,11 @@
 	</div>
 	<div data-type="Group" data-name="Groupe 819" id="Groupe_819_ebh" class="input-container">
 		<label for="Couriel">Couriel</label>
-		<input type="email" name="couriel" placeholder="Votre adresse email">
+		<input type="email" name="courriel" placeholder="Votre adresse email">
```

Comment:
This might induce a regression as the field name is used by SpringBoot to map the form to the JSon Bean